Templates/ScriptCanvasNode/Template/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+ly_add_target(
+    NAME ${Name}.Static STATIC
```

Comment:
This is missing the copyright blob
```cmake
# {BEGIN_LICENSE}
# Copyright (c) Contributors to the Open 3D Engine Project.
# For complete copyright and license terms please see the LICENSE at the root of this distribution.
#
# SPDX-License-Identifier: Apache-2.0 OR MIT
#
# {END_LICENSE}

```

```suggestion
ly_add_target(
    NAME ${Name}.Object OBJECT
```

Second note: This is probably better of being an OBJECT target, so that the file is added as an object file that gets built as part any existing STATIC, SHARED or MODULE library

Here is the CMake documentation about [OBJECT libraries](https://cmake.org/cmake/help/latest/command/add_library.html#object-libraries)

Reply:
Also one benefit of Object libraries, is that dead-stripping doesn't occur.

Which means in theory the registration of the Node can happen in the `${Name}.cpp` file, which could remove the step of a user needing to update their Gem's system component.

Reply:
This is excellent, I'll try it, thank you!